lib/text/ssa_text_parser.js

```diff
@@ -0,0 +1,352 @@
+      const ccsColor = SsaTextParser.parseSsaColor_(color);
+      if (ccsColor) {
+        cue.color = ccsColor;
+      }
+    }
+    const backgroundColor = style['BackColour'];
+    if (backgroundColor) {
+      const cssBackgroundColor = SsaTextParser.parseSsaColor_(backgroundColor);
+      if (cssBackgroundColor) {
+        cue.backgroundColor = cssBackgroundColor;
+      }
+    }
+    const bold = style['Bold'];
+    if (bold) {
+      cue.fontWeight = Cue.fontWeight.BOLD;
+    }
+    const italic = style['Italic'];
+    if (italic) {
+      cue.fontStyle = Cue.fontStyle.ITALIC;
+    }
+    const underline = style['Underline'];
+    if (underline) {
+      cue.textDecoration.push(Cue.textDecoration.UNDERLINE);
+    }
+    const letterSpacing = style['Spacing'];
+    if (letterSpacing) {
+      cue.letterSpacing = letterSpacing + 'px';
+    }
+    const alignment = style['Alignment'];
+    if (alignment) {
+      const alignmentInt = parseInt(alignment, 10);
+      switch (alignmentInt) {
+        case 1:
+          cue.displayAlign = Cue.displayAlign.AFTER;
+          cue.textAlign = Cue.textAlign.START;
+          break;
+        case 2:
+          cue.displayAlign = Cue.displayAlign.AFTER;
+          cue.textAlign = Cue.textAlign.CENTER;
+          break;
+        case 3:
+          cue.displayAlign = Cue.displayAlign.AFTER;
+          cue.textAlign = Cue.textAlign.END;
+          break;
+        case 5:
+          cue.displayAlign = Cue.displayAlign.BEFORE;
+          cue.textAlign = Cue.textAlign.START;
+          break;
+        case 6:
+          cue.displayAlign = Cue.displayAlign.BEFORE;
+          cue.textAlign = Cue.textAlign.CENTER;
+          break;
+        case 7:
+          cue.displayAlign = Cue.displayAlign.BEFORE;
+          cue.textAlign = Cue.textAlign.END;
+          break;
+        case 9:
+          cue.displayAlign = Cue.displayAlign.CENTER;
+          cue.textAlign = Cue.textAlign.START;
+          break;
+        case 10:
+          cue.displayAlign = Cue.displayAlign.CENTER;
+          cue.textAlign = Cue.textAlign.CENTER;
+          break;
+        case 11:
+          cue.displayAlign = Cue.displayAlign.CENTER;
+          cue.textAlign = Cue.textAlign.END;
+          break;
+      }
+    }
+    const opacity = style['AlphaLevel'];
+    if (opacity) {
+      cue.opacity = parseFloat(opacity);
+    }
+  }
+
+  /**
+   * Parses a SSA color .
+   *
+   * @param {string} colorString
+   * @return {?string}
+   * @private
+   */
+  static parseSsaColor_(colorString) {
+    // The SSA V4+ color can be represented in hex (&HAABBGGRR) or in decimal
+    // format (byte order AABBGGRR) and in both cases the alpha channel's
+    // value needs to be inverted as in case of SSA the 0xFF alpha value means
+    // transparent and 0x00 means opaque
+    /** @type {number} */
+    let abgr = -1;
+    try {
+      if (colorString.startsWith('&H')) {
+        // See: https://bugs.chromium.org/p/v8/issues/detail?id=2869
```

Comment:
Since you are just passing to `parseInt`, that issue doesn't seem relevant.  That issue is that `abgrString` will keep the whole buffer alive even if the parent is not referenced anymore.  Since this is a short-lived string, you don't need to workaround it here.

Reply:
I changed the parseSsaColor_ function, so now there are not problem.